base.mako

```diff
@@ -68,5 +68,6 @@
             ga('create', 'UA-104682927-1', 'auto');
             ga('send', 'pageview');
         </script>
+        <script src="assets/addCopyToBadging.js" type="text/javascript"></script>
```

Comment:
let's just call this `copyable.js` -- also I think in html5 the `type="text/javascript"` is assumed and can be removed

Reply:
done